src/test/OptionSettlement.t.sol

```diff
@@ -593,6 +621,88 @@ contract OptionSettlementTest is Test, NFTreceiver {
             emit FeeSwept(tokens[i], engine.feeTo(), expectedFees[i] - 1); // sweeps 1 wei less as gas optimization
         }
 
+        // When fees are swept
+        engine.sweepFees(tokens);
```

Comment:
Thanks for adding this. Surprised the test wasn't failing before

Reply:
Think I just added the comment. This test was calling sweepFees() 👍

Reply:
Ah see your other comment. Yeah just some diff weirdness